Stripe/UIViewController+BottomSheet.swift

```diff
@@ -13,9 +13,28 @@ extension UIViewController {
     func presentAsBottomSheet(
         _ viewControllerToPresent: BottomSheetPresentable,
         appearance: PaymentSheet.Appearance,
-        completion: (() -> ())? = nil
+        completion: (() -> Void)? = nil
```

Comment:
this change was introduced by the formatter, not sure if you'd like to keep this, don't have any preference